docs/package-management.md

```diff
@@ -267,3 +267,127 @@ The advantage of this approach —as opposed to vendoring the JavaScript package
 inside the bindings— is that it gives users of the bindings complete flexibility
 over the way these JavaScript packages are downloaded and bundled.
 
+## Finding and using Melange compatible packages
+
+### Opam packages
```

Comment:
```suggestion
### OPAM packages
```

Reply:
@anmonteiro we have been using `opam` all over, because that is how it is written in their doc page. https://opam.ocaml.org/

Reply:
ah fair point. I wasn't aware.